### ISSUE TITLE
Spotter colours

### ISSUE THREAD

**Issue by AnirudhG07** [catppuccin/yazi#14]
The recent update has spotter added, please add colors for it in the repository for yazi.
Thanks!

**Comment by AnirudhG07** [comment]
Yes, alright. Thanks

**Comment by AnirudhG07** [comment]
Just a reminder, to please edit the theme.toml file for yazi v0.4

**Comment by uncenter** [comment]
I've updated the rest, but Yazi itself doesn't support theming the spotter yet: https://github.com/sxyazi/yazi/pull/2002.

**Comment by AnirudhG07** [comment]
Lol.

**Comment by uncenter** [comment]
>  Since the spotter follows the default color settings, i think it should? But you must have looked into it, so not doubting u either.
I don't think it does, I'm not even sure how it is themed if it isn't even in the preset theme files. Anyway, lmk if anything else is unthemed, but I'll get to the spotter as soon as I can after that PR is merged.

**Comment by AnirudhG07** [comment]
Alright. I am again closing this issue since I got my answer. Thanks

**Comment by uncenter** [comment]
It doesn't, those are the default theme colors.

**Comment by AnirudhG07** [comment]
Does your spotter color theme match?

**Comment by AnirudhG07** [comment]
I see. still it looks nice mostly.
